editor/effects.js

```diff
@@ -20,9 +20,30 @@ import {
 	getCurrentPostType,
 	getBlocks,
 	getPostEdits,
+	isCleanNewPost,
 } from './selectors';
 
+let originalDocumentTitle;
+
+function populateDocumentTitle( title, isCleanNew = false ) {
+	if ( ! title.trim() ) {
```

Comment:
It's unnecessary because `document.title` has its whitespace automatically normalized: https://html.spec.whatwg.org/multipage/dom.html#document.title